src/python/grpcio/grpc/_adapter/_types.py

```diff
@@ -165,7 +165,7 @@ class OpArgs(collections.namedtuple(
     message (bytes): Only valid if type == OpType.SEND_MESSAGE, else is None.
     status (Status): Only valid if type == OpType.SEND_STATUS_FROM_SERVER, else
       is None.
-    write_flags (int): a bit OR'ing of 0 or more OpWriteFlags values.
+    flags (int): a bit OR'ing of 0 or more OpWriteFlags values.
```

Comment:
"A bit-ORing ..." or "A bitwise ORing of ..."? I just accidentally misread this as describing a one-bit integer. :-P